src/loaders/ObjectLoader.js

```diff
@@ -564,13 +564,19 @@ Object.assign( ObjectLoader.prototype, {
 				}
 
 				if ( data.format !== undefined ) texture.format = data.format;
+				if ( data.type !== undefined ) texture.type = data.type;
+				if ( data.encoding ) texture.encoding = data.encoding;
```

Comment:
Why not using this pattern?
```js
if ( data.encoding !== undefined ) texture.encoding = data.encoding;
```

Reply:
Oops, I guess I was too sleepy when I was making this PR. Thanks.